server/src/main/java/com/objectcomputing/checkins/services/file/FileServicesImpl.java

```diff
@@ -93,10 +93,6 @@ public FileServicesImpl(GoogleApiAccess googleApiAccess,
                 CheckIn checkIn = checkInServices.read(checkInID);
                 validate(checkIn == null, String.format("Unable to find checkin record with id %s", checkInID));
 
-                if(!isAdmin) {
-                    validate((!currentUser.getId().equals(checkIn.getTeamMemberId()) && !currentUser.getId().equals(checkIn.getPdlId())), "You are not authorized to perform this operation");
```

Comment:
Should we update this to call `access granted` method instead of removing validation? That way, only the users with right permission will be able to access the resource.

Reply:
the checkinservice is doing this